freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -65,12 +74,75 @@ def handle(self, event):
         self.log_info(
             "Orignial nvrs of build in the advisory #{0} are: {1}".format(
                 db_event.search_key, " ".join(original_nvrs)))
+        # Get images by nvrs and then get their digests
+        original_images_digests = set(self._pyxis.get_digests_by_nvrs(original_nvrs))
```

Comment:
@apaplaus I'm now realizing that you are getting `original_nvrs` from the event you define in `db_event = Event.get_or_create_from_event(db.session, event)`. The `original_nvrs` will not be defined though since that event did not previously exist and you need to search Freshmaker for the original NVRs associated with operator/operand images that were rebuilt and shipped as part of the `BotasErrataShippedEvent`.

This bug was part of the code previous to this PR though, so could you please file a ticket so it can be addressed in a separate PR?